packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -32,43 +72,103 @@ export function FormatFile({ file, onDelete }: FormatFileProps) {
   }
 
   const style = imageSource ? { backgroundImage: `url(${imageSource})` } : {};
+  const imageBlockStyle = isComplete
+    ? styles.imageBlock
+    : classNames(styles.imageBlock, styles.imageBlockOverlay);
 
   return (
-    <div className={styles.formatFile}>
-      <div className={styles.imageBlock} style={style} data-testid="imageBlock">
-        {!imageSource && (
-          <div className={styles.icon}>
-            <Icon name={iconName} />
+    <>
+      {display === "file" && (
```

Comment:
This conditional seems to do a lot going on. I wonder if we can break these conditionals off into two internal components. One to handle the file preview and the other to handle the thumbnail

Reply:
After looking at the element structure my gut tells me the conditionals can be change to just pass different classNames, style and testIds that could be determined before and set as `consts`